assets/js/calypsoify.js

```diff
@@ -55,7 +55,27 @@
 	/**
 	 * Move page actions to action header.
 	 */
-	$( '.page-title-action, .add-new-h2' ).appendTo( '#action-header .action-header__actions' );
+	$( document ).ready( function() {
+		const $actions = $( '.page-title-action, .add-new-h2' );
+		if ( ! $actions.is( 'button' ) ) {
+			const buttons = [];
+			$actions.each( function( index, anAction ) {
+				const { className, href, text } = anAction;
+				const button = $( '<button/>' ,{
+					class: 'button button-primary',
+					text,
```

Comment:
I think this would fail as well in IE11 since object shorthand is ES6

Reply:
You're right! This was fixed in the commit 7b5ef73